src/batch-client/index.ts

```diff
@@ -1346,7 +1341,14 @@ export class ZimbraBatchClient {
 
 			for (const pref in prefs) {
 				if (CASTING_PREFS.indexOf(pref) !== -1) {
-					prefs[pref] = typeof prefs[pref] === 'string' ? castArray(prefs[pref]) : prefs[pref];
+					prefs[pref] =
+						typeof prefs[pref] === 'string'
```

Comment:
can we simplify logic by
array = [].concat(array || string)
we just need to bifurcate logic of undefined

Reply:
Addressed.